dsbulk/Dockerfile

```diff
@@ -0,0 +1,55 @@
+# Copyright DataStax, Inc, 2017
+#   Please review the included LICENSE file for more information.
+#
+
+FROM dse-base
+
+MAINTAINER "DataStax, Inc <info@datastax.com>"
+
+ARG VERSION=1.3.3
+ARG URL_PREFIX=http://localhost
+ARG DSBULK_TARBALL=dsbulk-${VERSION}.tar.gz
+ARG DSBULK_DOWNLOAD_URL=${URL_PREFIX}/${DSBULK_TARBALL}
+
+
+ENV DSBULK_HOME /dsbulk
+
+RUN set -x \
```

Comment:
not sure if really need to create any user inside the image.

Reply:
I prefer not to run under root, but need to test - if it will be easier to run as specific user if inside Docker we have root